styles.css

```diff
@@ -0,0 +1,42 @@
+@import url('https://fonts.googleapis.com/css?family=Barlow+Condensed|Barlow:600|Rasa:300,500');
+
+body {
+    background-image: linear-gradient(rgba(158, 58, 58, 0.8), rgba(206, 136, 136, 0.7)),url(https://img.freepik.com/free-photo/horizontal-view-fresh-delicious-juice-glass-with-orange-lime-tube-dark-background_140725-140821.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    font-family: 'Barlow', sans-serif;
+    color: white;
+    align-items: center;
+}
+
+#header {
+    margin:auto;
```

Comment:
I would like to suggest considering the usage of grid and flex displays as a means of achieving a responsive webpage design. Implementing these techniques can help ensure that your content is displayed optimally across a range of devices and screen sizes. For further guidance on this topic, you may find it helpful to review the concepts of **Flexbox** and **Grid layout** provided in these links: 

- [Flexbox](https://github.com/microverseinc/curriculum-html-css/blob/main/flexbox.md)
- [Grid Layout](https://github.com/microverseinc/curriculum-html-css/blob/main/grid.md)